tests/fsbv.lisp

```diff
@@ -137,7 +137,7 @@
   5)
 
 ;;; returning struct with bitfield member (bug #1474631)
-(defbitfield (struct-bitfield :unsigned-int)
+(defbitfield (struct-bitfield :int)
```

Comment:
hrm... i wrongly assumed at one point that enum's in C are always signed.

got rid of this change and recorded a commit to clean up the enum base type mapping.